src/TasksModule/Components/TasksList/TasksList.tsx

```diff
@@ -1,7 +1,65 @@
-import React from 'react'
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { baseUrl } from "../../../Constants/Components/Urls";
+import DeleteModal from "../../../SharedMoudule/Components/DeleteModal/DeleteModal";
 
 export default function TasksList() {
+  const navigate = useNavigate();
+  const token = localStorage.getItem("userToken");
+  const [openDeleteModal, setOpenDeleteModal] = useState(false);
+  const [currentTaskId, setCurrentTaskId] = useState(null);
+  const [Tasks, setTasks] = useState(null);
+  const closeModal = () => {
+    setOpenDeleteModal(false);
+  };
+
+  const getdata = async () => {
```

Comment:
I suggest renaming this to `getTasks` as `getData` is too generic and doesn't indicate exactly what this function does.